src/Entity/ListBuilder/EdgeEntityListBuilder.php

```diff
@@ -126,10 +126,20 @@ protected function buildEntityIdQuery(): QueryInterface {
   public function render() {
     $settings = $this->getDisplaySettings();
     if ($this->usingDisplayType(static::VIEW_MODE_DISPLAY_TYPE)) {
-      return $this->renderUsingViewMode($settings['view_mode']);
+      $build = $this->renderUsingViewMode($settings['view_mode']);
     }
+    else {
+      $build = parent::render();
+    }
+
+    // Add cache contexts.
+    $build['#cache'] = [
+      'contexts' => $this->entityType->getListCacheContexts(),
+      'tags' => $this->entityType->getListCacheTags(),
+      'max-age' => $this->getCacheMaxAge(),
```

Comment:
Isn't the entity type's default cache should be considered here or it should be the same as what is stored in the entity type specific config (by default)?

Reply:
Not sure what you mean here. Can you point me to `"what is stored in the entity type specific config (by default)"`? Thanks.

Reply:
Ohh, so there is no `getListCacheMaxAge()` on Drupal entity types by default, probably because of the same reason: https://github.com/apigee/apigee-edge-drupal/issues/430#issuecomment-649539244

so probably on Apigee Edge entity types, we should have a `getListCacheMaxAge()` method that returns the applicable max-age for an entity type specific list builders and it should retrieve the value from the related config.

Reply:
Yeah. Only `getListCacheContexts` and `getListCacheTags` on `EntityInterface`.

We could implement a `getListCacheMaxAge` in `EdgeEntityInterface` but not all entity types defines a `list_builder`.

I think we can leave it on the base `EdgeEntityListBuilder` class.